README.md

```diff
@@ -73,7 +76,19 @@ You can find the Public API Key in your [dashboard](https://dashboard.fingerprin
 > [!NOTE]
 > There is also `release` variant for the internal use. It's very similar to `releaseLocalSign`, but signs the app with another signature and disables the stubbing functionality described above.
```

Comment:
@Alexey-Verkhovsky - What does "internal use" mean?

Reply:
This means for testing, and not for publishing. Release builds are ready for publishing, they don't contain debug symbols and performance is better for them. I changed the line, I hope now it sounds better.